spec/controllers/self_assessments_controller_spec.rb

```diff
@@ -10,7 +10,8 @@
 
   def valid_attributes
     {
-       response: "My Response"
+       response: "My Response",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.